contrib/node/src/python/pants/contrib/node/tasks/node_build.py

```diff
@@ -69,7 +69,9 @@ def execute(self):
       for vt in invalidation_check.all_vts:
         target = vt.target
         node_installed_path = node_paths.node_path(target)
-
+        # Relying on NodePaths not having a node resolver defined for NodeThriftLibrary targets
+        if not node_installed_path:
```

Comment:
All targets should have a node_paths, otherwise it should fail

Reply:
Fixed, now we have a separate node_paths_local for the generated targets.